trust-store-trust-policy-specification.md

```diff
@@ -135,6 +162,19 @@ Value descriptions
 - **`warn`**: This means implementation MUST perform the validation and if validation fails(because of any reason) the implementation MUST log an error and MUST NOT fail validation.
 - **`skip`**: This means implementation MUST NOT perform the validation.
 
+#### Scopes Constraints
+
+- Each trust policy MUST contain scope property and the scope collection MUST contain at least one value.
+- The scope MUST contain one of the following:
+  - List of one or more fully qualified repository URIs. The repository URI MUST NOT contain the asterisk character `*`.
+  - A single value with one asterisk character `*`. The scope with `*` value is called global scope. The trust policy with global scope applies to all the artifacts. There can only be one trust policy that uses a global scope.
+- For a given artifact there MUST be only one applicable trust policy, with the exception of trust policy with global scope.
+- For a given artifact, if there is no applicable trust policy then Notary v2 MUST consider the artifact as untrusted and fail signature verification.
+- The scope MUST NOT support reference expansion i.e. URIs must be fully qualified. E.g. the scope should be `docker.io/library/registry` rather than `registry`.
+- Evaluation order of trust policies:
+  1. *Exact match*: If there exists a trust policy whose scope contains the artifact's repository URI then the aforementioned policy MUST be used for signature evaluation. Otherwise, continue to the next step.
```

Comment:
What if there exists two or more trust policies whose scope contains the repository URI of the artifact?

Reply:
> For a given artifact there MUST be only one applicable trust policy, with the exception of trust policy with global scope.

Oh, I see the constraints. That means the client should do a scope validation on loading / updating the trust policies.